src/lightning_app/cli/commands/connection.py

```diff
@@ -14,20 +15,24 @@
 from lightning_app.utilities.log import get_logfile
 from lightning_app.utilities.network import LightningClient
 
+_HOME = os.path.expanduser("~")
+_PPID = str(psutil.Process(os.getpid()).ppid())
```

Comment:
I don't think we install psutil by default. Might need to be added to base requirement if that's the only way to get the parent pid.

Reply:
Yes, let's keep psutil. This is always useful to clean up process stuffs.